src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -739,6 +742,13 @@ void UserInfoPopup::setData(const QString &name,
         this->updateLatestMessages();
     }
     // If we're opening by ID, this will be called as soon as we get the information from twitch
+
+    if (auto *tc = dynamic_cast<TwitchChannel *>(this->channel_.get());
+        tc == nullptr)
+    {
```

Comment:
this will also hide button from /mentions or search where it works as expected,
could be changed to check if channel type is `TwitchWhispers` or `TwitchLive`